common/lib/xmodule/xmodule/graders.py

```diff
@@ -344,28 +343,28 @@ def __init__(
         self.starting_index = starting_index
         self.hide_average = hide_average
 
-    def grade(self, grade_sheet, generate_random_scores=False):
-        def total_with_drops(breakdown, drop_count):
```

Comment:
**Note to reviewers:** Moved this internal method out to an accessible method so it can be called to compute an assignment-type percentage (without going through the entire grading process).